programs/keeper/Keeper.cpp

```diff
@@ -365,7 +366,7 @@ try
     }
 
     /// Initialize keeper RAFT. Do nothing if no keeper_server in config.
-    global_context->initializeKeeperDispatcher(/* start_async = */ true);
+    global_context->initializeKeeperDispatcher(/* start_async = */ false);
```

Comment:
The `Keeper` class can be used either by `clickhouse-keeper` executable or by `clickhouse-keeper` symlink. It's not used by `clickhouse-server`, so asynchronous starting is not required here. See also the [assert](https://github.com/ClickHouse/ClickHouse/blob/a6861ea2f5aedc67c9cc5da16a835af8f00faea3/src/Interpreters/Context.cpp#L2831) in `Context::initializeKeeperDispatcher()`.